src/util/serverUtils.js

```diff
@@ -92,10 +93,8 @@ export function registerExtensionEvents(server) {
 	server.ext([{
 		type: 'onRequest',
 		method: onRequestExtension,
-	}, {
-		type: 'onPreResponse',
-		method: onPreResponseExtension,
 	}]);
+	server.on('response', logResponse);
```

Comment:
turns out the full _response_ headers are not available in `onPreResponse`, so we have to hook into this event in order to correctly log the full response info